### PR TITLE
[ISSUE #4814] Migrate from fastjson 1.2.83 to fastjson2

### PR DESCRIPTION
**Pull request by cnzakii**
### Motivation
[Enhancement] Migrate from fastjson 1.2.83 to fastjson2 [#4814](https://github.com/apache/eventmesh/issues/4814)

### Modifications
 remove the `fastjson:1.2.83` and add the `fastjson2:2.0.48`

### Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by Pil0tXia** [comment]
plz check ci~

File: eventmesh-connectors/eventmesh-connector-prometheus/build.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ dependencies {
     implementation project(":eventmesh-openconnect:eventmesh-openconnect-java")
     implementation 'org.apache.httpcomponents:httpclient'
     implementation 'com.github.rholder:guava-retrying'
-    implementation 'com.alibaba:fastjson'
+    implementation 'com.alibaba:fastjson2'
     compileOnly 'org.projectlombok:lombok'
```
[the remark this answers is not in the document]
Comment (cnzakii) [reply eventmesh-connectors/eventmesh-connector-prometheus/build.gradle:23]: Thank you very much for helping me pinpoint the issue. I did see an error indication in IntelliJ IDEA, but I didn't notice this particular problem, so I ended up overlooking it (IDEA sometimes has bugs related to dependencies 😭).

**Review by Pil0tXia — DISMISSED** [review]
LGTM

**Comment by cnzakii** [comment]
@Pil0tXia 

I encountered an error during the[License Check](https://github.com/cnzakii/eventmesh/actions/runs/8616377630/job/23613845462#logs)，it appears to be due to：



This dependency was brought in through `org.apache.rocketmq:rocketmq-client:4.9.5`:

**Comment by Pil0tXia** [comment]
@cnzakii 

You may add it in known-dependencies.

**Comment by cnzakii** [comment]
> @cnzakii
> 
> You may add it in known-dependencies.
@Pil0tXia 

Completed. It's an effective measure.

**Review by Pil0tXia — APPROVED** [review]

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**